pkgs/tools/admin/mycli/default.nix

```diff
@@ -1,28 +1,31 @@
 { lib
 , python
```

Comment:
If mycli is compatible with python3, please replace this by `python3`. Otherwise, by `python2`.

Reply:
I tried using python3, the tests fail like this:

```sh-session
=================================== FAILURES ===================================
________________________________ test_list_dsn _________________________________

    def test_list_dsn():
        runner = CliRunner()
        with NamedTemporaryFile(mode="w") as myclirc:
            myclirc.write(dedent("""\
                [alias_dsn]
                test = mysql://test/test
                [main]
                log_file = /tmp/mycli.test.log
                """))
            myclirc.flush()
            args = ['--list-dsn', '--myclirc', myclirc.name]
            result = runner.invoke(cli, args=args)
>           assert result.output == "test\n"
E           AssertionError: assert '' == 'test\n'
E             + test

args       = ['--list-dsn', '--myclirc', '/build/tmpbldvl9dg']
myclirc    = <tempfile._TemporaryFileWrapper object at 0x7fffedb39da0>
result     = <Result RuntimeError('Click will abort further execution because Python 3 was configured to use ASCII as encoding for ... suitable UTF-8\nlocales were discovered.  This most likely requires resolving\nby reconfiguring the locale system.',)>
runner     = <click.testing.CliRunner object at 0x7fffedb39dd8>

test/test_main.py:287: AssertionError
==== 1 failed, 186 passed, 59 skipped, 1 xfailed, 1 xpassed in 5.42 seconds ====
builder for '/nix/store/m4vvpfnpgnlsa265cagv1hq99yfh5fs8-mycli-1.17.0.drv' failed with exit code 1
e
```

I put `python2` there for now, we can revisit in a separate PR.